joomla/joomla-cms
# [5.1] Adminmenu: Toggle for duplicate menus

Pull Request for Issue #32934 .

### Summary of Changes

Same as the 4.4-dev PR #43192 , but this here is for 5.1-dev.

As the modified file has been refactored in the 5.1-dev branch with PR #42784 , the change from PR #43192 cannot be merged up from 4.4-dev into 5.1-dev as it is.

Therefore I've created this PR here which does the same change as PR #43192 but at a different place due to the refactoring.

### Testing Instructions

See #43192 :
1. Create a second menu module in the administrator, assign it to the menu position and select the "Joomla Main Menu" preset for the module.
2. Click on the toggle button to close the sidebar.
3. Click on the "Content" menu item in the second module.

### Actual result BEFORE applying this Pull Request

The sidebar stays closed.

### Expected result AFTER applying this Pull Request

The sidebar opens, like for the menu entries in the first menu module.

### Link to documentations
Please select:
